torchvision/models/_api.py

```diff
@@ -203,19 +204,45 @@ def wrapper(fn: Callable[..., M]) -> Callable[..., M]:
     return wrapper
 
 
-def list_models(module: Optional[ModuleType] = None) -> List[str]:
+def list_models(
+    module: Optional[ModuleType] = None,
+    include_filters: Union[Iterable[str], str, None] = None,
+    exclude_filters: Union[Iterable[str], str, None] = None,
+) -> List[str]:
     """
     Returns a list with the names of registered models.
 
     Args:
         module (ModuleType, optional): The module from which we want to extract the available models.
+        include_filters (str or Iterable[str], optional): Filter(s) for including the models from the set of all models.
+            Filters are passed to fnmatch to match Unix shell-style wildcards. In case of many filters, the results is
```

Comment:
Since it's not immediately clear what `fnmatch` is, it helps to add a link
(same below)

```suggestion
            Filters are passed to `fnmatch <https://docs.python.org/3/library/fnmatch.html>`__ to match Unix shell-style wildcards. In case of many filters, the results is
```